<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComputeCurvatureDialog</class>
 <widget class="QDialog" name="ComputeCurvatureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compute Curvature</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="mapList"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Position attribute :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_positionAttribute">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Normal attribute :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_normalAttribute">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Computed attributes</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Kmax :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Existing attribute :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="combo_KmaxAttribute">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>New attribute :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="KmaxAttributeName"/>
        </item>
       </layout>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="check_KmaxCreateVBO">
        <property name="text">
         <string>create VBO</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>kmax :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Existing attribute :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="combo_kmaxAttribute">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>New attribute :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="kmaxAttributeName"/>
        </item>
       </layout>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="check_kmaxCreateVBO">
        <property name="text">
         <string>create VBO</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Kmin :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Existing attribute :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="combo_KminAttribute">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>New attribute :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="KminAttributeName"/>
        </item>
       </layout>
      </item>
      <item row="2" column="3">
       <widget class="QCheckBox" name="check_KminCreateVBO">
        <property name="text">
         <string>create VBO</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_13">
        <property name="font">
         <font>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>kmin :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Existing attribute :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="combo_kminAttribute">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>New attribute :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="kminAttributeName"/>
        </item>
       </layout>
      </item>
      <item row="3" column="3">
       <widget class="QCheckBox" name="check_kminCreateVBO">
        <property name="text">
         <string>create VBO</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_15">
        <property name="font">
         <font>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Knormal :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Existing attribute :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="combo_KnormalAttribute">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>New attribute :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="KnormalAttributeName"/>
        </item>
       </layout>
      </item>
      <item row="4" column="3">
       <widget class="QCheckBox" name="check_KnormalCreateVBO">
        <property name="text">
         <string>create VBO</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_apply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_ok</sender>
   <signal>clicked()</signal>
   <receiver>ComputeCurvatureDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_cancel</sender>
   <signal>clicked()</signal>
   <receiver>ComputeCurvatureDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>156</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
